tasks/push-to-cdn/push-to-cdn.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-to-cdn
+  labels:
+    app.kubernetes.io/version: "4.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push content to Red Hat's CDN
+  params:
+    - name: releasePlanAdmission
+      type: string
+      description: The namespaced name of the ReleasePlanAdmission
+      default: "rhtap-releng-tenant/rpa" # This needs to change later
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: Workspace to save the CR jsons to
```

Comment:
Maybe this is leftover copied code, but I don't see any CR jsons being saved here

Reply:
correct, cp/ps error, I have updated the description, ty!